packages/react/src/components/Footer/__tests__/Footer.test.js

```diff
@@ -63,5 +63,8 @@ xdescribe('<Footer />', () => {
     expect(footer.querySelectorAll('.bx--legal-nav__list-item')).toHaveLength(
       MOCK_DATA.footerThin.length + 1
     );
+
+    console.log(footer.querySelectorAll('.bx--footer-nav-group'));
```

Comment:
@RaphaelAmadeu probably need to remove this console log

Reply:
This was removed on a latest PR